bundles/org.openhab.binding.dbquery/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.dbquery</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: DBQuery Binding</name>
+
+  <properties>
+    <bnd.importpackage>
+      !javax.annotation;!android.*,!com.android.*,!com.google.appengine.*,!dalvik.system,!kotlin.*,!kotlinx.*,!org.conscrypt,!sun.security.ssl,!org.apache.harmony.*,!org.apache.http.*,!rx.*,!org.msgpack.*
+    </bnd.importpackage>
+  </properties>
+
+  <dependencies>
+    <!-- influxdb-client-java -->
+    <dependency>
+      <groupId>com.influxdb</groupId>
+      <artifactId>influxdb-client-java</artifactId>
+      <version>1.6.0</version>
+    </dependency>
+    <dependency>
+      <artifactId>influxdb-client-core</artifactId>
+      <groupId>com.influxdb</groupId>
+      <version>1.6.0</version>
+    </dependency>
+    <dependency>
+      <artifactId>converter-gson</artifactId>
+      <groupId>com.squareup.retrofit2</groupId>
+      <version>2.5.0</version>
+    </dependency>
+    <dependency>
+      <artifactId>converter-scalars</artifactId>
+      <groupId>com.squareup.retrofit2</groupId>
+      <version>2.5.0</version>
+    </dependency>
+    <dependency> <!-- also used for querydb library -->
+      <artifactId>gson</artifactId>
+      <groupId>com.google.code.gson</groupId>
+      <version>2.8.5</version>
+    </dependency>
+    <dependency>
+      <artifactId>gson-fire</artifactId>
+      <groupId>io.gsonfire</groupId>
+      <version>1.8.0</version>
+    </dependency>
+    <dependency>
+      <artifactId>okio</artifactId>
+      <groupId>com.squareup.okio</groupId>
+      <version>1.17.3</version>
+    </dependency>
+    <dependency>
+      <artifactId>commons-csv</artifactId>
+      <groupId>org.apache.commons</groupId>
+      <version>1.6</version>
+    </dependency>
```

Comment:
Can you remove this library and use Java 11 methods instead? See https://github.com/openhab/openhab-addons/issues/7722

Reply:
@Skinah I'm sorry but it's not possible to do it in a safe way.
The addon doesn't use commons code at all, but the influxdb driver uses it and it's not documented as an optional dependency.
Also, I don't know if it applied to that dependency because up to my knowledge  there isn't any support for csv parsing in Java 11

Reply:
@fwolter can you comment on this one please? thanks.

Reply:
Seems to be a reasonable usage of Apache commons in this case.